autoload/pastery.vim

```diff
@@ -0,0 +1,58 @@
+let s:pyver = has("python3") ? "python3" : "python"
+
+function pastery#PasteCode(start, end)
+    exec s:pyver "PasteryPaste(" a:start "," a:end ")"
+endfunction
+
+function pastery#PasteFile()
+    exec s:pyver "PasteryPaste(title='" expand('%:t') "')"
+endfunction
+
+exec s:pyver . "<< EOF"
```

Comment:
I just moved the Python code from `plugin/pastery.vim` to here. I also moved the imports out of `PasteryPaste` because the 2159cb3e3867faa2abaef1514cfa68851895aed3 tweak isn't needed anymore.